src/YarpUtilities/src/RosPublisher.cpp

```diff
@@ -96,13 +111,21 @@ class RosPublisher::Impl
     
     bool publishTF{false}; /**< flag to enable publishing transforms to YARP transform server */    
     bool initialized{false}; /**< flag to chekc if the publisher was initialized */
+    
+    // placeholder variables
+    yarp::sig::Matrix pose; /**< placeholder variable for publishing transform data*/
+    std::vector<double> jointStateZero; /**< placeholder variable for zero joint state data*/
+    
+    std::chrono::nanoseconds timeInNanoSeconds; /**< chrono nanoseconds conversion from YARP to ROS timestamp*/
```

Comment:
This is unused, I will remove it.

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-framework/pull/156/commits/1f927f2d77a8f4cb629753f29fe92cc4861af96d